azure-pipelines.yml

```diff
@@ -16,6 +17,14 @@ jobs:
   pool:
     vmImage: 'Ubuntu 16.04'
   steps:
+  - script: |
+      echo TRAVIS: $(TRAVIS)
+      echo TRAVIS_BRANCH: $(TRAVIS_BRANCH)
+      echo TRAVIS_REPO_SLUG: $(TRAVIS_REPO_SLUG)
+      echo TRAVIS_PULL_REQUEST: $(TRAVIS_PULL_REQUEST)
+      echo TRAVIS_PULL_REQUEST_SLUG: $(TRAVIS_PULL_REQUEST_SLUG)
+      echo TRAVIS_PULL_REQUEST_SHA: $(TRAVIS_PULL_REQUEST_SHA)
+      echo PR_ONLY: $(PR_ONLY)
```

Comment:
I know you already merged this (Sorry that I didn't see it earlier), but I definitely recommend putting a `displayName` property on each of your script steps. It just helps to know what each step does when you're looking at it on the DevOps page.